docs/src/proposals/multiple-gossip-identities.md

```diff
@@ -0,0 +1,61 @@
+# Multiple Gossip Identities
+
+The current design of gossip requires that a single validator ID is run by a
+single node. If more than one machine runs at the same time they will overwrite
+each others CRDS values. Ideally, a validator might want to be able to run
+multiple nodes in gossip that represent the same ID.
```

Comment:
I'd be good to discuss how voting and the leader role works between validators in a group.   If this is to be enabled by a hook into external software then the API into that mechanism should be discussed here